packages/manager/src/features/NodeBalancers/NodeBalancerCreate.test.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+
+import { renderWithTheme } from 'src/utilities/testHelpers';
+
+import NodeBalancerCreate from './NodeBalancerCreate';
+
+// Note: see nodeblaancers-create-in-complex-form.spec.ts for an e2e test of this flow
+describe('NodeBalancerCreate unit tests', () => {
```

Comment:
```suggestion
describe('NodeBalancerCreate', () => {
```

(generally we just have the component name here)